GameServer/Extensions.cs

```diff
@@ -97,45 +154,56 @@ public bool ContainsKey(TKey key)
             }
         }
     }
-    public class CustomConvert
+    public static class CustomConvert
     {
-        public static TeamId toTeamId(int i)
+        public static TeamId ToTeamId(int i)
         {
-            switch (i)
+            var dic = new Dictionary<int, TeamId>
+            {
+                { 0, TeamId.TEAM_BLUE },
```

Comment:
This looks wrong, as in, you probably want only one mapping for the team ID, not two.

Created #239 from this, you don't have to change anything